papers/oliver_beckstein/oliver_beckstein.rst

```diff
@@ -93,6 +93,32 @@ The user interface and modular design work equally well in complex scripted work
 MDAnalysis is available in source form under the GNU General Public License v2 from GitHub https://github.com/MDAnalysis/mdanalysis and PyPi;  conda packages are also available. The documentation is extensive http://docs.mdanalysis.org including an introductory tutorial http://www.mdanalysis.org/MDAnalysisTutorial/ and a very friendly and welcoming user and developer community. 
 
 
+New data Structures
+-------------------
+
+Originally MDAnalysis followed a strict object oriented approach with a separate instance of an Atom object for each particle in the simulation data.
+The AtomGroup then simply stored its contents as a list of these Atom instances.
+With simulation data commonly containing 10\ :sup:`6` particles
+this solution did not scale well and so
+recently this design was overhauled to improve the scalability of MDAnalysis.
+
+With all Atoms having the same property fields it was possible to store this information as a series of NumPy arrays,
+with a single array for each property.
+The AtomGroup can then store its contents as an array of integers
+which represent the address in the property arrays which correspond to the data for the stored Atoms.
+
+Overall this approach means that the same number of Python objects are created for each Universe,
+with the data size only changing the size of the arrays.
+This translated into a much smaller memory footprint (BENCHMARK HERE)
+highlighting the memory cost of millions of simple Python objects.
```

Comment:
The result of this new design **note: we said design before so lets keep using that word** is that every new Universe object creates a fixed number of Python objects. This translated into a much smaller memory footprint **Benchmark results/plot**, highlighting the memory cost of millions of simple Python objects.